src/main/webapp/app/exam/manage/exams/exam-update.component.ts

```diff
@@ -131,7 +130,11 @@ export class ExamUpdateComponent implements OnInit {
     get isValidExamStudentReviewEnd(): boolean {
         // allow instructors to set examStudentReviewEnd later
         if (!this.exam.examStudentReviewEnd) {
-            return true;
+            if (!this.exam.examStudentReviewStart || !this.exam.examStudentReviewStart.isValid()) {
```

Comment:
The code below has nothing to do with the comment above, or does it?

Reply:
No, it is adapted in order to be more meaningful now.